src/main/java/net/mcreator/ui/dialogs/TextureMappingDialog.java

```diff
@@ -67,22 +68,21 @@ public TextureMappingDialog(Map<String, TexturedModel.TextureMapping> currentSta
 			return null;
 		}
 
-		MCreatorDialog d = new MCreatorDialog(mcreator, "Texture mappings for model", true);
+		MCreatorDialog d = new MCreatorDialog(mcreator, L10N.t("dialog.textures_mapping.title_for_model"), true);
 
 		JTabbedPane pane = new JTabbedPane();
 		pane.setTabLayoutPolicy(JTabbedPane.SCROLL_TAB_LAYOUT);
 		pane.setForeground(Color.white);
 
 		if (supportMultiple) {
-			pane.addTab("Add new texture mapping", null);
+			pane.addTab(L10N.t("dialog.textures_mapping.add_new"), null);
 			pane.setTabComponentAt(0, new JLabel(UIRES.get("16px.add.gif")));
 			Set<String> finalTexturesList = texturesList;
 			pane.addChangeListener(e -> {
 				if (pane.getSelectedIndex() == 0) { // new texture mapping
 					pane.setSelectedIndex(1);
-					String mapping = VOptionPane.showInputDialog(mcreator, "<html>Enter your texture mapping name.<br>"
-									+ "This name is used to identify your set of textures when<br>"
-									+ "selecting a texture mapping for your model.", "New texture mapping", null,
+					String mapping = VOptionPane.showInputDialog(mcreator, L10N.t("dialog.textures_mapping.enter_name"),
+							L10N.t("dialog.textures_mapping.new"), null,

```

Comment:
You used the "_title"/"_message" suffix on all the other dialogue entries but this one. ".title"/".message" would be better, but you use underscore suffixes everywhere so at least stick to one naming convention.

Reply:
Done, I renamed  `dialog.textures_mapping.enter_name` into `dialog.textures_mapping.enter_name_message` and `dialog.textures_mapping.new` into `dialog.textures_mapping.enter_name_title`